src/magicbox/QueryProcessor.ts

```diff
@@ -51,16 +53,20 @@ export class QueryProcessor<T> {
     }
   }
 
+  private get cleanOrderedResults(): T[] {
```

Comment:
Starting the name with a verb makes the function sound like a `void` method. Would `orderedResults` work?

Reply:
clean is a verb and an adjective 😏